x-pack/plugins/security_solution/public/overview/containers/overview_cti_links/index.tsx

```diff
@@ -4,92 +4,67 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { useState, useEffect, useCallback } from 'react';
-import type { SavedObjectAttributes } from '@kbn/securitysolution-io-ts-alerting-types';
+import { useEffect, useMemo } from 'react';
 import type { TiDataSources } from './use_ti_data_sources';
-import type { LinkPanelListItem } from '../../components/link_panel';
 import { useKibana } from '../../../common/lib/kibana';
+import { getDashboardsByTagIds } from '../../../common/containers/dashboards/api';
+import { getTagsByName } from '../../../common/containers/tags/api';
+import { useFetch, REQUEST_NAMES } from '../../../common/hooks/use_fetch';
+import { SecurityPageName } from '../../../../common/constants';
+import { useGetSecuritySolutionUrl } from '../../../common/components/link_to';
 
-const TAG_REQUEST_BODY_SEARCH = 'threat intel';
-export const TAG_REQUEST_BODY = {
-  type: 'tag',
-  search: TAG_REQUEST_BODY_SEARCH,
-  searchFields: ['name'],
+const CTI_TAG_NAME = 'threat intel';
```

Comment:
Found the tag I have matches this name, so didn't change it. @elastic/security-detections-response  Could you please confirm this is the right tag.

<img width="1503" alt="Screenshot 2023-05-11 at 09 22 39" src="https://github.com/elastic/kibana/assets/6295984/d800a91e-4bbd-4a2c-aa83-b7074b6020c5">